Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -181,6 +181,7 @@ export function runQTest(args: QTestArguments): Result {
         untrackedScopes: [
             d`d:/data`,
             d`d:/app`,
+            d`${sandboxDir.pathRoot}/$Recycle.Bin`,

```

Comment:
Is this because QTest performs operations on the recycle bin?

If it is customer (test code or code under test) then that test should actually do the work.

Reply:
The customer test case in question perfoms a Win32 Delete operation in one of the QTests. DFA flagging tracks the recycle bin as of today, which results in the test being flagged.

Reply:
So then technically the customer should declare the dependency...  the Qtest ask can't aggregate all possible file pattern of customers. Since recycle bin is sort of potentially generic for tests I'm okay, but technically this belongs in the customers' set of dependencies/untracked space...

Reply:
The QTest SDK has a user specifiable hook for adding additional dependencies but there isn't one for untracking directories. I think it's ok to special case this since we really don't want/expect additional per-test untracking. The majority of accesses are in the QTest sandbox which is already untracked. In short, I think the change is fine

Reply:
K, sounds good